src/components/validatedInput.jsx

```diff
@@ -29,13 +30,13 @@ const HourInput = React.createClass({
         return 'error';
     },
 
-    validationMessage () {
-        const {hour, isValid} = this.props.value;
+    getValidationMessage () {
+        const {text, isValid} = this.props.value;
 
-        if (isValid || !hour) {
+        if (isValid || !text) {
```

Comment:
`!text` should probably be checked only if there's a 'required' props or something like that